config/gateway/gateway-plugin/gateway-plugin.yaml

```diff
@@ -31,6 +36,24 @@ spec:
       labels:
         app: gateway-plugins
     spec:
+      affinity:
+        podAntiAffinity: # pods are placed on different nodes
+           requiredDuringSchedulingIgnoredDuringExecution:
```

Comment:
if this is required, single node scenario can not serve 2 replicas. Prod env commonly has more than 1 node, but I am a little bit concern user raise such problem again.
- either update the docs to tell user you need >= 2 nodes in the cluster for HA purpose
- change to `preferred`

Reply:
Changed to preferred.